docker/Dockerfile

```diff
@@ -1,22 +1,25 @@
-FROM debian:jessie
+FROM frolvlad/alpine-glibc
 
-ENV FSM_VERSION 0.2.0
+MAINTAINER Jannik Kolodziej <docker@jkolodziej.de>
 
-RUN apt-get update && \
-    apt-get -y install unzip wget ca-certificates && \
-    apt-get clean
+ENV FACTORIO_VERSION=0.12.31 \
+    MANAGER_VERSION=0.2.0 \
+    ADMIN_PASSWORD=
 
-RUN wget https://github.com/MajorMJR/factorio-server-manager/releases/download/$FSM_VERSION/factorio-server-manager-linux-x64.zip -P /tmp/
-RUN unzip /tmp/factorio-server-manager-linux-x64.zip -d /opt/
+VOLUME /opt/factorio/saves /opt/factorio/mods /security
 
-WORKDIR /opt/factorio-server
+RUN apk add --no-cache curl tar unzip nginx
 
-VOLUME ["/factorio"]
+WORKDIR /opt/
+ADD "init.sh" "/opt/init.sh"
+ADD "nginx.conf" "/etc/nginx/nginx.conf"
 
-#Expose UI Port
-EXPOSE 8080
+RUN curl -s -L -S -k https://www.factorio.com/get-download/$FACTORIO_VERSION/headless/linux64 -o /tmp/factorio_$FACTORIO_VERSION.tar.gz && \
+    tar zxf /tmp/factorio_$FACTORIO_VERSION.tar.gz && \
+    rm /tmp/factorio_$FACTORIO_VERSION.tag.gz && \
```

Comment:
oh wow. that was actually quite a big size increase for the layeer and I totally missed it. Thank you :)

Fixed